scripts/build/build.mjs

```diff
@@ -126,7 +126,7 @@ function handleError(error) {
 async function preparePackage() {
   const packageJson = await readJson(path.join(PROJECT_ROOT, "package.json"));
   packageJson.bin = "./bin-prettier.js";
-  packageJson.engines.node = ">=10.13.0";
+  // packageJson.engines.node = ">=10.13.0";
```

Comment:
Why commented out?

Reply:
I can remove it if you prefer.

Reply:
Ah sorry, I just want to know the reason.

Reply:
Maybe we need a different node version for production in future, we can comment it back.

Reply:
I got it